src/shared/components/EmptyFallback/EmptyFallback.styles.ts

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled'
+
+import { Text } from '@/shared/components'
```

Comment:
Components inside `shared` should use relative imports
```suggestion
import { Text } from '../Text'
```